stock_picking_product_generate_sequence/README.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
```

Comment:
[Major] If the module will be contributed to the OCA you don't need to generate the README. This is done by the OCA automatically based on the contents of the Readme folder, and the readme won't have info about braintec but about OCA

Reply:
thx, I removed README file

Reply:
The README.rst file is still there. Or am I wrong?

Reply:
sorry, you're right, I didn't delete the file.